marimo/_convert/markdown/markdown.py

```diff
@@ -161,20 +180,23 @@ def get_cell_config_from_tag(tag: Element, **defaults: bool) -> CellConfig:
     return CellConfig.from_dict(extracted_attrs)
 
 
+T = TypeVar("T")
+
+
 # TODO: Consider upstreaming some logic such that this isn't such a terrible
 # hack. At some point rewriting / overriding the markdown parser would be a
 # better idea than all these little work arounds.
 @dataclass
-class SafeWrap:
-    app: App
+class SafeWrap(Generic[T]):
```

Comment:
Been meaning to move this file, so this is good. This might break quarto, but will check- generic typing seems more robust than the previous case

I think the test suite is pretty robust, so if they pass (esp, snapshot) I'm good with this